frontend/src/app/components/op-context-menu/icon-triggered-context-menu/icon-triggered-context-menu.component.ts

```diff
@@ -28,14 +28,9 @@
 
 import {ChangeDetectorRef, Component, ElementRef, Injector, Input} from '@angular/core';
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
-import {AuthorisationService} from 'core-app/modules/common/model-auth/model-auth.service';
 import {OpContextMenuTrigger} from 'core-components/op-context-menu/handlers/op-context-menu-trigger.directive';
 import {OPContextMenuService} from 'core-components/op-context-menu/op-context-menu.service';
 import {OpModalService} from "core-components/op-modals/op-modal.service";
-import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/isolated-query-space";
-import {BoardListComponent} from "core-app/modules/boards/board/board-list/board-list.component";
-import {Board} from "core-app/modules/boards/board/board";
-import {BoardActionsRegistryService} from "core-app/modules/boards/board/board-actions/board-actions-registry.service";
 import {OpContextMenuItem} from "core-components/op-context-menu/op-context-menu.types";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c9bc013cc27eb3bb23aea9c85aaedb5b868ef858/frontend/src/app/components/op-context-menu/icon-triggered-context-menu/icon-triggered-context-menu.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>